package/AUTHORS

```diff
@@ -236,6 +236,7 @@ Chronological list of authors
   2024
   - Aditya Keshari
   - Philipp Stärk
+  - SumairaAhmad
```

Comment:
Most contributors put their real name here, which typically contains spaces. But you can also put your GitHub handle here if you do not want to put your real name. In this case it would be `nataliyah123` as far as I can tell.